app/org/iatoki/judgels/raguel/services/ThreadPostService.java

```diff
@@ -16,6 +16,10 @@
 
     Map<String, String> getThreadPostsJidToUserJidMap(List<String> threadPostJids);
 
+    Map<String, Long> getUserJidToPostCountMap(List<String> userJids);
```

Comment:
I don't see any use of this method outside the Service, if that is the case, it should be private and not specified in interface.